src/cli.rs

```diff
@@ -550,6 +550,10 @@ pub struct Opt {
     /// Text to display in front of a renamed file path.
     pub file_renamed_label: String,
 
+    #[structopt(long = "right-arrow", default_value = "⟶  ")]
```

Comment:
Note to self: there are two trailing spaces in this default value in order to match what was previously hard-coded in various places (whether or not the previous treatment was correct is a separate issue).